Core/Core/Helpers/Open.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Diagnostics;
+
+namespace Speckle.Core.Helpers;
+
+public static class Open
+{
+  public static void Url(string url)
+  {
+    var psi = new ProcessStartInfo { FileName = url, UseShellExecute = true };
+    try
+    {
+      Process.Start(psi);
+    }
+#pragma warning disable CA1031
+    catch
+#pragma warning restore CA1031
+    {
+      psi.UseShellExecute = false;
+      Process.Start(psi);
```

Comment:
What this for?

Reply:
In case we can't shell execute for backwards compatibility and cross platformness.  The false for .NET Core was because not all platforms like it.  Like Mac/Linux.